caddy/caddymain/run.go

```diff
@@ -50,6 +51,7 @@ func init() {
 	flag.StringVar(&disabledMetrics, "disabled-metrics", "", "Comma-separated list of telemetry metrics to disable")
 	flag.StringVar(&conf, "conf", "", "Caddyfile to load (default \""+caddy.DefaultConfigFile+"\")")
 	flag.StringVar(&cpu, "cpu", "100%", "CPU cap")
+	flag.StringVar(&envFile, "env-file", "", "File with environment variables to load")
```

Comment:
I think I would prefer just `env` as the name of the cli directive.

Reply:
Hm...i believe env-file is better name because "env" parameter is used to provide single env value usually. 

Docker run for example:
```
 % docker run --help 2>&1|grep env
  -e, --env list                       Set environment variables
      --env-file list                  Read in a file of environment variables
```

Reply:
I can not see the use case for using `-env` to supply env vars to caddy. It makes sense for docker because the container does not have access to any env vars unless those passed to it; which is not the case with caddy. 

I vote for `-env` as well.

Reply:
Oh, ok. I didn't thought about it. I will rename it and renew PR.